mon-pix/tests/acceptance/module/verify-qcu_test.js

```diff
@@ -0,0 +1,77 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { visit } from '@1024pix/ember-testing-library';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { click } from '@ember/test-helpers';
+
+module('Acceptance | Module | Routes | verifyQcu', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  test('can validate my QCU answer', async function (assert) {
+    // given
+    const qcu1 = server.create('qcu', {
+      id: 'elementId-1',
+      type: 'qcus',
+      instruction: 'instruction',
+      proposals: [
+        { id: 'qcu-1-proposal-1', content: 'I am the wrong answer!' },
+        { id: 'qcu-1-proposal-2', content: 'I am the right answer!' },
+      ],
+    });
+    const qcu2 = server.create('qcu', {
+      id: 'elementId-2',
+      type: 'qcus',
+      instruction: 'instruction',
+      proposals: [
+        { id: 'qcu-2-proposal-1', content: 'Vrai' },
+        { id: 'qcu-2-proposal-2', content: 'Faux' },
+      ],
+    });
+
+    const grain = server.create('grain', {
+      id: 'grainId',
+      title: 'title',
+      elements: [qcu1, qcu2],
+    });
+
+    server.create('module', {
+      id: 'bien-ecrire-son-adresse-mail',
+      title: 'Bien écrire son adresse mail',
+      grains: [grain],
+    });
+
+    server.create('correction-response', {
+      id: 'elementId-1',
+      feedback: "Bravo ! C'est la bonne réponse.",
+      status: 'ok',
+      solution: 'qcu-1-proposal-2',
```

Comment:
Pour être raccord avec le model 
```suggestion
      solutionId: 'qcu-1-proposal-2'
```